eng/build.ps1

```diff
@@ -0,0 +1,38 @@
+[CmdletBinding(PositionalBinding=$false)]
+Param(
+  [string] $ArchGroup,
+  [switch] $Release,
+  [switch] $c,
+  [switch] $Restore,
+  [switch] $Test,
+  [switch] $Pack,
+  [switch] $SignPackages,
+  [switch] $Publish,
+  [Parameter(ValueFromRemainingArguments=$true)][String[]]$ExtraArgs
+)
+
+$defaultargs = "-build -restore -warnaserror:0"
+
+foreach ($argument in $PSBoundParameters.Keys)
+{
+  switch($argument)
+  {
+    "Debug"        { $arguments += "-configuration Debug " }
+    "Release"      { $arguments += "-configuration Release " }
+    "ExtraArgs"    { $arguments += $ExtraArgs }
+    "c"            { $arguments += "-rebuild " }
+    "Restore"      { $arguments += "" }
+    "Test"         { $arguments += "" }
+    "Pack"         { $arguments += "" }
+    "SignPackages" { $arguments += "/p:SignPackages=true " }
+    "Publish"      { $arguments += "/p:PublishPackages=true /p:BuildRestore=false /p:BuildManaged=false /p:BuildTests=false /p:BuildPackages=false " }
```

Comment:
I was hoping that we could just call the Publish target directly on build.proj passing using eng/common/MSBuild is there a reason you didn't go that route?

Reply:
I didn't see a distinction between `Publish` and the other standard options (`Test`, `Pack`, etc.). Is there a downside to having a `Publish` option here?

Reply:
I just seems very odd to go through the Build target and disable everything. It seems pretty fragile so I think we should just call msbuild on publish.msbuild and remove the Publish target from build.proj.

Reply:
Fair enough, I'll change it